Assets/Mirror/Runtime/NetworkReader.cs

```diff
@@ -186,16 +186,14 @@ public ArraySegment<byte> ReadBytesSegment(int count)
         // null support, see NetworkWriter
         public byte[] ReadBytesAndSize()
         {
-            if (ReadBoolean())
-            {
-                return ReadBytes(checked((int)ReadPackedUInt32()));
-            }
-            return null;
+            uint count = ReadPackedUInt32();
```

Comment:
@MichalPetryka I meant here, in the actual reader+writer code. would be useful to have your examples there on how this works